src/core/renderers/webgl/filters/Filter.js

```diff
@@ -85,8 +85,17 @@ export default class Filter
          * If enabled is true the filter is applied, if false it will not.
          *
          * @member {boolean}
+         * @default true
          */
         this.enabled = true;
+
+        /**
+         * If enabled, pixi will fit the filter area into boundaries, it helps for shaders with mapping
+         *
+         * @member {boolean}
+         * @default true
+         */
+        this.fitToDestination = true;
```

Comment:
Not crazy about the name of the property. It's a little confusing and verbose. Isn't this more accurate as `fitToSource`? Or maybe, simply: `autoFit`?